test/svgGenerator.js

```diff
@@ -0,0 +1,66 @@
+const {ethers} = require("hardhat");
+const {expect} = require('chai');
+
+const getSvgMintConfog = (minted, maxMintable, totalCollateralValue) => {
+    return {
+        vaultAddress: "0x1234567890123456789012345678901234567890",
+        minted: ethers.BigNumber.from(ethers.utils.parseUnits(minted, 18)),
+        maxMintable: ethers.BigNumber.from(ethers.utils.parseUnits(maxMintable, 18),),
+        totalCollateralValue: ethers.BigNumber.from(ethers.utils.parseUnits(totalCollateralValue, 18)),
```

Comment:
i think `ethers.BigNumber.from` is superfluous here as `parseUnits` returns a big number. you can remove

Reply:
referenced in: [8952a3b](https://github.com/the-standard/smart-vault/pull/1/commits/8952a3bb6e8a561e3b59a080b27a15986337f884)